src/main/kotlin/no/nav/k9/los/nyoppgavestyring/mottak/oppgave/OppgaveV3Repository.kt

```diff
@@ -37,7 +37,12 @@ class OppgaveV3Repository(
         val oppgaveId = nyOppgaveversjon(oppgave, nyVersjon, tx)
         lagreFeltverdier(oppgaveId, oppgave, tx)
 
-        AktivOppgaveRepository.ajourholdAktivOppgave(oppgave, nyVersjon, tx)
+        val ignorerForKøer = OppgaveV3.gjelderFRISINN(oppgave) && oppgave.oppgavetype.eksternId == "k9sak"
```

Comment:
Frisinn er ikke i los for noen av systemene. Det blir behandlet av en annen enhet som ikke bruker los

Reply:
Nice, har fjernet k9sak-sjekken nå